packages/cosmic-swingset/src/sim-chain.js

```diff
@@ -39,12 +39,11 @@ async function makeMapStorage(file) {
   };
 
   let obj = {};
-  try {
+  await (async () => {
     content = await fs.promises.readFile(file);
     obj = JSON.parse(content);
-  } catch (e) {
-    return map;
-  }
+  })().catch(() => map);
```

Comment:
This is subtly different than the original code.  Could you please make it more obvious that this `catch` is a no-op?
```suggestion
  })().catch(() => {});
```

Reply:
I missed that but see it now. This is a bigger difference than I intended to allow myself. If the original code's catch runs, it returns `map` early and the 
```js
Object.entries(obj).forEach(([k, v]) => map.set(k, importMailbox(v)));
```
line is not run. In this PR, that line is run regardless. How concerned should I be?

Reply:
Done. But I'll leave this unresolved until I hear back.

Reply:
I'm not too concerned with the extra line being evaluated.  The bookkeeping to detect the exception would be a distraction where really I just wanted to ensure that a read or parse failure would result in an empty map.

Reply:
Or maybe the most faithful pattern is straightforward:
```
   await (async () => {
     ...
     return obj;
   )().then(obj => Object.entries(obj).forEach(...), () => { /* do nothing */ });
   return map;
```

Reply:
Resolving because I'm nitpicking and don't want to delay seeing this PR land!

Reply:
Doh! Done.